build/Dockerfile

```diff
@@ -6,20 +6,22 @@ COPY . .
 
 RUN ATTACHER_TAG=libbpf make build
 
-FROM registry.access.redhat.com/ubi9-minimal:9.2
-RUN microdnf -y update
+FROM registry.access.redhat.com/ubi9
+RUN yum -y update
+RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+RUN if [ $(uname -i) == "x86_64" ]; then yum install -y cpuid; fi
```

Comment:
installing `epel-release` and `cpuid` in the main image causes image size to bloat thats why i kept the installation in the base image, and just copied the binary.

Reply:
as cpuid just for x86, we don't have too much option here.  As cpuid is x86 specific, we can't copy the cpuid binary from base to main image, as for no x86 image, there is no cpuid rpm package support.
hence, one of the option is that we make some script specifically base on cpu arch. I am free and open to any workable option considering with multiple arch.

Reply:
since `cpuid` is not available on other archs, there  is no other option than what you suggest.